frontend/packages/ceph-storage-plugin/src/__mocks__/independent-mode-dashboards.ts

```diff
@@ -0,0 +1,67 @@
+export const DashboardItems = {
+  watchURL: {
+    url: 'foo',
+  },
+  stopWatchURL: 'foobar',
+  watchPrometheus: {
+    query: 'fooQuery',
+  },
+  stopWatchPrometheusQuery: {
+    query: 'fooQuery2',
+  },
+  watchAlerts: '',
+  stopWatchAlerts: '',
+  urlResults: 'foo',
+  prometheusResults: {
+    // eslint-disable-next-line no-new-func
+    getIn: new Function(),
+  },
+  notificationAlerts: 'foo',
+  watchK8sResource: 'foo',
+  stopWatchK8sResource: 'foo',
+};
+
+export const dropDownItems = {
+  Pods: 'Pods',
+  Projects: 'Projects',
+  'Storage Classes': 'Storage Classes',
+};
+
+export const headerLink =
+  'topk(20, (sum(kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"})) by (namespace)))';
+
+export const detailsCardData = {
```

Comment:
Why not simply move `detailsCardData.resources` into `dashboardData.resources` and remove the `detailsCardData` export?

This way, you'll have a single source of test data (`dashboardData`) that can be used in all relevant tests.